protocol-tokio/src/protocol/codec/handshake.rs

```diff
@@ -17,7 +17,7 @@ use super::{MessageCode, MessageType};
 pub struct ProtocolMagic(u32);
 impl Default for ProtocolMagic {
     fn default() -> Self {
-        ProtocolMagic(764824073)
+        ProtocolMagic(1097911063)
```

Comment:
I wonder if we could have other `ProtocolMagic` values, or is the whole thing just an unnecessarily glorified const?

Reply:
we should have only one (maybe to) const protocol magic, mainnet and staging
the other ones are testing networks.

Reply:
So this is not just the protocol magic, but also determines the chain.

Reply:
We should take that value as a parameter, I think I'll need to extend network configuration, so this value will be passed by the user. I'd like to do that in the separate commit unless it's a blocker.